src/style.css

```
@@ -1,3 +1,74 @@
+:root {
+  --text-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #FFF;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header .header__a {
+  width: 40px;
+  height: 40px;
+}
+
+.nav .nav__list {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  list-style: none;
+  position: relative;
+}
+
+.nav .nav__list .nav__item:not(:last-child) {
+  margin: 0 20px 0 0;
+}
+
+.nav .nav__list .nav__item a {
```

Need to use the classes selector
```suggestion
.nav__link {
```